### PR TITLE
February 10 Release Notes

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes here to communicate to the maintainers what you changed and why.

## Checklist for contributors

Before submitting this PR, please make sure:

* [ ] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [ ] You have signed the [Axway CLA](https://cla.axway.com/)
* [ ] You have verified the technical accuracy of your change
* [ ] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [ ] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | e1fdfb1 |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-263.amplify-central.opendocs-builder.pcloud.axway.int |

**Review by robertengaxway — APPROVED** [review]
I approve

**Review by cbordier — APPROVED** [review]
I approve the changes

File: content/en/docs/amplify_relnotes/20230210_amplify.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+* **Publish from Service registry to the Marketplace** - The image from the API Service is used as a reference for the Asset and the Product but it is not displayed in the Marketplace.
+
+### Business insights
+
+The following enhancements and bug fixes are available in this release.
+
+#### Feature updates for Business insights
+
+* **Business Insights shows the organization and consumer team** - Application and Subscription screens display the consumer org and the consumer team who generated the traffic. For traffic related to provider organization, they are displayed as "Internal".
+* **Subscriptions data can be exported as a CSV file** - Provider can export to a CSV file the data they see on Business Insights / Subscriptions using the 3 dots beside the time filter.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20230210_amplify.md:140]: A provider can export the subscription records in Business Insights to a CSV file by using the ellipses menu located next to the time filter.

File: content/en/docs/amplify_relnotes/20230210_amplify.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+* **Multi select capability** - Provider can select multiple teams in the Provider Engagement screen.
+
+#### Bug fixes for Business Insights
+
+* **Large number of application showing "Unknown"** - The transactions that are not initiated from a Marketplace subscription are now removed from the Application and Subscription screens.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20230210_amplify.md:145]: Large number of applications showing "Unknown"

File: content/en/docs/amplify_relnotes/20230210_amplify.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+
+The following enhancements and bug fixes are available in this release.
+
+#### Feature updates for Marketplace
+
+No new feature in this release.
+
+#### Bug fixes for Marketplace
+
+* **Deprecated / archived plan is still available for subscription** - The deprecate/archived plans are no more available when consumer initiate a subscription.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20230210_amplify.md:159]: Consumers can no longer subscribe to a plan that is either deprecated or archived.

File: content/en/docs/amplify_relnotes/20230210_amplify.md [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+* **Corrupted product does not show the warning icon in all marketplaces** - The corrupted product are displayed with the warning icon in all marketplaces the product is deployed on.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20230210_amplify.md:160]: A corrupted product now shows a warning icon in all Marketplaces it is published into.

**Review by georgiana-caras — CHANGES_REQUESTED** [review]
Requested some changes

**Pull request merged**